app/editor/src/components/row/Row.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { CSSProperties } from 'styled-components';
+import * as styled from './RowStyled';
+
+export interface IRowProps {
+  /**
+   * place desired nodes in row
+   */
+  children?: React.ReactNode;
+  /**
+   * additional styling flexibility
+   */
+  style?: CSSProperties;
+}
+
+export const Row: React.FC<IRowProps> = ({ children, style }) => {
```

Comment:
Just inherit from HTMLDivElement props.  Don't have custom properties interface unless you're adding new features.